install_config/syncing_groups_with_ldap.adoc

```diff
@@ -176,46 +184,59 @@ OpenShift. Your files must contain one unique group identifier per line.
 ====
 
 ----
-$ oadm groups sync --whitelist=whitelist.txt --sync-config=config.yaml --confirm
-$ oadm groups sync --blacklist=blacklist.txt --sync-config=config.yaml --confirm
-$ oadm groups sync group_unique_idenitifer --sync-config=config.yaml --confirm
-$ oadm groups sync group_unique_idenitifer   \
-                           --whitelist=whitelist.txt \
-                           --blacklist=blacklist.txt \
-                           --sync-config=config.yaml --confirm
-$ oadm groups sync --type=openshift --whitelist=whitelist.txt --sync-config=config.yaml --confirm
+$ oadm groups sync --whitelist=<whitelist_file> --sync-config=config.yaml --confirm
+$ oadm groups sync --blacklist=<blacklist_file> --sync-config=config.yaml --confirm
+$ oadm groups sync <group_unique_identifier> --sync-config=config.yaml --confirm
+$ oadm groups sync <group_unique_identifier> \
+                           --whitelist=<whitelist_file> \
+                           --blacklist=<blacklist_file> \
+                           --sync-config=config.yaml \
+                           --confirm
+$ oadm groups sync --type=openshift \
+                           --whitelist=<whitelist_file> \
+                           --sync-config=config.yaml \
+                           --confirm
 ----
 
+[[running-a-group-pruning-job]]
 == Running a Group Pruning Job
-An administrator can also choose to remove groups from OpenShift records if the records on the LDAP
-server that created them are no longer present. The prune job will accept the same sync
-configuration file and white- or black-lists as used for the sync job.
 
-For instance, if groups had previously been syncheonized from LDAP using some `config.yaml`, and some of those
-groups no longer existed on the LDAP server, the following command would determine which groups in OpenShift
-corresponded to the deleted groups in LDAP and then remove them from OpenShift.
+An administrator can also choose to remove groups from OpenShift records if the
+records on the LDAP server that created them are no longer present. The prune
+job will accept the same sync configuration file and white- or black-lists as
+used for the sync job.
+
+For example, if groups had previously been synchronized from LDAP using some
+*_config.yaml_* file, and some of those groups no longer existed on the LDAP
+server, the following command would determine which groups in OpenShift
+corresponded to the deleted groups in LDAP and then remove them from OpenShift:
+
 ----
-$ openshift groups prune --sync-config=config.yaml --confirm
+$ oadm groups prune --sync-config=config.yaml --confirm
```

Comment:
@stevekuznetsov Noticed this left-over `openshift` command after merging the last PR.